python/ray/autoscaler/updater.py

```diff
@@ -335,7 +358,23 @@ def run(self,
             timeout=timeout,
             exit_on_fail=exit_on_fail,
             port_forward=None,
-            with_output=False)
+            with_output=False,
+            ssh_options_override=ssh_options_override)
+
+    def check_docker_installed(self):
+        install_commands = [
```

Comment:
very nit: I'd put this just above the error message so it's clear that it's only used there

Reply:
Done!